jujugui/static/gui/src/app/app.js

```diff
@@ -652,12 +651,8 @@ YUI.add('juju-gui', function(Y) {
                               (this.get('socket_url') || this.get('sandbox')));
         if (isNotGISFSandbox || isNotConnected) {
           this.env.connect();
-          if (this.controllerAPI) {
-            // We won't have a controller API connection in Juju 1 or in
```

Comment:
Why is this comment no longer true? Isn't it still possible that we are on Juju 1 and this.controllerAPI is undefined?